### ISSUE TITLE
mask r-cnn

### ISSUE THREAD

**Issue by elifbykl** [jaspereb/FasterRCNNTutorial#5]
Hi Jasper,

I want to apply the mask r-cnn. Are there changes in the implementation steps? For example, while training dataset is being created, I need to train with object boundaries rather than being rectangular. Which tool can I use for this?

**Comment by jaspereb** [comment]
Yes, you will need to annotate the object boundaries in the training set. The linked tutorial suggests using VGG Image annotator for labelling. LabelBox is also very good but more aimed at large collaborative projects.

https://engineering.matterport.com/splash-of-color-instance-segmentation-with-mask-r-cnn-and-tensorflow-7c761e238b46

**Comment by elifbykl** [comment]
@jaspereb thank you for your answer. I put the JPEG images in the JPEGImages folder as if you were working. VGGImageAnnotater saved annotations as .json format. Do I need to put this file in the Annotations folder or do I need to apply a different conversion to .json format?

**Comment by jaspereb** [comment]
How you set up the file structure will depend on the dataset preprocessing script that you use (or write your own). The one for fasterRCNN will not work because it expects bounding boxes, not regions. This is really nothing to do with this tutorial, so you are better off following a mask-rcnn tutorial.
